### ISSUE TITLE
Plans Next: Remove the wrapped components exported in `plans-grid-next/src/index.tsx`

### ISSUE THREAD

**Issue by chriskmnds** [Automattic/wp-calypso#87047]
As of https://github.com/Automattic/wp-calypso/pull/86730, we no longer need the `WrappedFeaturesGrid` and `WrappedComparisonGrid` components, since the underlying components are both in function form now.

We can address this a bit later, once we finalize things a bit more.

**Comment by southp** [comment]
@chriskmnds I'm interested to looking into this further. My understanding for the end result is that we will directly export `FeaturesGrid` and `ComparisonGrid` here: https://github.com/Automattic/wp-calypso/blob/trunk/packages/plans-grid-next/src/index.tsx#L161. Does that mean `PlansGridContextProvider` will also be removed? Just want to make sure that I won't attempt to overly clean things up :)

**Comment by chriskmnds** [comment]
> @chriskmnds I'm interested to looking into this further. My understanding for the end result is that we will directly export `FeaturesGrid` and `ComparisonGrid` here: https://github.com/Automattic/wp-calypso/blob/trunk/packages/plans-grid-next/src/index.tsx#L161. Does that mean `PlansGridContextProvider` will also be removed? Just want to make sure that I won't attempt to overly clean things up :)

@southp Yeah, that's the idea - move the `WrappedComparisonGrid`/`WrappedFeaturesGrid` to the respective folders and only have imports/exports from the main index. As you like though. There may be value in keeping these here, although it makes them available to adding more code/logic - which should probably live in the other folders.
